extensions/api/data-management/asset/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 Diego Gomez
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *   Diego Gomez - Initial API and Implementation
+ */
+
+val infoModelVersion: String by project
+val rsApi: String by project
+val jerseyVersion: String by project
+
+plugins {
+    `java-library`
+}
+
+dependencies {
+    api(project(":spi"))
```

Comment:
here, `:spi:core-spi` is sufficient

Reply:
I had to import also :spi:web-spi" for the WebService instance in in the method initialize() from the class AssetControllerServiceExtension (line 36) since it's not part of core-spi (at least from the current version)